src/bin/gpfdist/regress/output/gpfdist_old_ssl.source

```diff
@@ -1,3 +1,37 @@
+--
+-- GPFDISTS test cases
+--
+-- start_ignore
+drop external table if exists gpfdist_ssl_start;
+NOTICE:  table "gpfdist_ssl_start" does not exist, skipping
+drop external table if exists gpfdist_ssl_stop;
+NOTICE:  table "gpfdist_ssl_stop" does not exist, skipping
+-- end_ignore
+-- --------------------------------------
```

Comment:
ignore content in output is redundant

Reply:
this is cherry pick. is it worth editing it?

Reply:
> this is cherry pick. is it worth editing it?

OK